### PR TITLE
NXP-29712: Fix CLID usage when installing a package in the Docker image

### PR DESCRIPTION
**Pull request by ataillefer**

### CODE REVIEW

**Review by kevinleturc — APPROVED** [review]

**Review by troger — APPROVED** [review]

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -57,7 +57,7 @@ fi
 if [ -n "$NUXEO_CLID" ]; then
   echo "ENTRYPOINT: Write NUXEO_CLID environment variable to /var/lib/nuxeo/instance.clid"
   # Replace -- by a carriage return
-  NUXEO_CLID="${NUXEO_CLID/--/\\n}"
+  NUXEO_CLID="${NUXEO_CLID//--/\\n}"
```
Comment (efge) [inline docker/docker-entrypoint.sh:60]: What is this supposed to do? This syntax with `//` is not documented

Comment (efge) [reply docker/docker-entrypoint.sh:60]: Ok sorry it is documented, I didn't read the bash doc carefully enough.

**Review by efge — APPROVED** [review]

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (ataillefer) [reply docker/docker-entrypoint.sh:60]: Nor the commit description :smile: https://github.com/nuxeo/nuxeo/pull/4346/commits/e73bf009fe19cc81f9c96c998448c8449e899ceb

Comment (efge) [reply docker/docker-entrypoint.sh:60]: Hm you mean the extended description hidden behind "..." in the PR pages...

**Pull request merged**